cmd/boot-script-service/default_api.go

```diff
@@ -450,6 +450,16 @@ func BootparametersPost(w http.ResponseWriter, r *http.Request) {
 			fmt.Sprintf("Bad Request: %s", err))
 		return
 	}
+	// Check that the xnames are valid
+	err = args.CheckXnames()
```

Comment:
(No change requested here, just commenting for posterity)

See comment in pkg/bssTypes/types.go on line 61. When a solution comes for the mentioned issue, this may change to be conditional in the future.